Extension/package.json

```diff
@@ -2373,21 +2373,26 @@
               "copyright",
               "date",
               "details",
-              "exception",
+              "exceptions",
               "invariant",
               "note",
+              "param",
               "pre",
               "post",
               "remark",
               "remarks",
               "result",
               "return",
+              "returns",
+              "retval",
               "since",
+              "tparam",
               "test",
               "todo",
               "version",
               "warning"
               ],
+          "default": [ "brief", "param", "tparam", "returns", "retval", "warning", "note" ],
```

Comment:
"brief" is not in the enum list.
"tparam" comes before "param"
"deprecated" and "exceptions" should be in this list somewhere after "retval"